packages/core/src/sass/app/_common/layout/app-layouts.scss

```diff
@@ -90,15 +90,14 @@
     font-size: 1.6rem;
     z-index: 9999;
     font-weight: bold;
+    gap: 2.4rem;
 
     &__icon {
-        width: 59px;
-        height: 64px;
+        width: 80px;
+        height: 62px;
```

Comment:
rem ?

Reply:
Resolved and also updated the height to the correct values 😃 ☝🏼